debugInterface.js

```diff
@@ -9,11 +11,7 @@ function Uint8ArrayToString (input) {
 }
 
 function Uint8ArrayToHexString (input) {
-  var ret = ''
-  for (var i = 0; i < input.length; i++) {
-    ret += input[i].toString(16)
-  }
-  return ret
+  return new Buffer(input).toString('hex')
```

Comment:
Buffers are just Uint8 arrays with some sugar. The problem the previous code is that it printed a single digit when it should have print two digets. For example it would print `1` when it need to print `01` since its repenting two nibbles